third_party/outline-tun2socks/index.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright 2023 The Outline Authors
```

Comment:
This should either live under `src/` or in `outline-go-tun2socks`.
Code in third_party should be minimal. Putting in outline-go-tun2socks may make sense, since it aligns with what we do on mobile.

Reply:
Yes, but then we need to publish it to npm first (maybe as a [private package](https://docs.npmjs.com/creating-and-publishing-private-packages)), otherwise our client cannot reference this package.

Reply:
No need to publish. You can npm install from github if you put in outline-go-tun2socks. But src/ works too